resdk/resources/base.py

```diff
@@ -235,6 +235,29 @@ def permissions(self):
 
         return self._permissions
 
+    def _list_holders_with_perm(self, perm):
+        """Get users/group/public names that have ``perm`` perm."""
+        holders = []
+        self.permissions.fetch()
+
+        for item in self.permissions._permissions:  # pylint: disable=protected-access
+            if perm in item['permissions']:
+                holders.append(item.get('name', 'Public'))
```

Comment:
It would be nice (and consistent) if you return `User` objects instead of just names.

Reply:
Good point. But returning the "full" user would require another request for each user... We can create user instances "partially", like in `BaseResolweReource.contributor`. But this can be tricky since User instance will be missing some of the attributes that it should have.